### DISCUSSION TITLE
Add `folke/noice.nvim` to improve UI

### DISCUSSION THREAD

**Discussion by vollowx** [ayamir/nvimdots#661] [Ideas]
### Feature description

Replace `lsp_signatures.nvim`, `fidget.nvim`, `wilder.nvim` and some other UI elements with this plugin.

**Comment by charliie-dev** [comment]
discussions is a better place for this kind of ideas!

**Comment by vollowx** [comment]
That's for sure! (this's the 1st time I use the feature 'discussion')

**Reply by aarnphm** [reply]
noice.nvim is very hacky, so probably users can use it if wanted. I don't think it fits with the main repo.

**Reply by charliie-dev** [reply]
> noice.nvim is very hacky, so probably users can use it if wanted. I don't think it fits with the main repo.

Agree on that. I tried `noice.nvim` few months ago. And I just can't bear making everything into floating windows. I not sure about `noice.nvim`'s current status, but the plugin replacement propose needs a solid implementation demo to gone further.

**Reply by aarnphm** [reply]
I think noice.nvim has some POC wrt to message routing and inspiration from fidget.nvim, which is nice. Other than that, the nui integration with floating windows is opinionated and not a one-size-fit-all.
